### ISSUE TITLE
[Bug] Couldn't use virtualenv_dir with Google Cloud Composer

### ISSUE THREAD

**Issue by anai-s** [astronomer/astronomer-cosmos#1172]
### Astronomer Cosmos Version

Other Astronomer Cosmos version (please specify below)

### If "Other Astronomer Cosmos version" selected, which one?

1.5.1

### dbt-core version

1.7.18

### Versions of dbt adapters

dbt-bigquery==1.7.4
dbt-adapters==1.4.0 (package in composer 2.9.1-airflow 2.9.1 image)

### LoadMode

AUTOMATIC

### ExecutionMode

VIRTUALENV

### InvocationMode

None

### airflow version

2.9.1

### Operating System

Windows 10

### Deployment

Google Cloud Composer

### Deployment details

1. PYPI Packages
- astronomer-cosmos==1.5.1
- dbt-bigquery==1.7.4

2. Composer Environment Configuration
- Composer 2 - Image composer-2.9.1-airflow-2.9.1
- location europe-west1
- others configurations are the default one

### What happened?

I expected to be able to test `virtualenv_dir` with Cloud Composer by reproducing the example dag here https://github.com/astronomer/astronomer-cosmos/blob/main/dev/dags/example_virtualenv.py but I got the error below

### Relevant log output

```shell
Broken DAG (dags/test_dag.py):
Traceback (most recent call last):
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/airflow/gcs/dags/test_dag.py", line 28, in <module>
    execution_config=ExecutionConfig(
                     ^^^^^^^^^^^^^^^^
TypeError: ExecutionConfig.__init__() got an unexpected keyword argument 'virtualenv_dir'
```

### How to reproduce

Upload the following file my_cosmos_dag.py into Composer DAG Folder

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

**Comment by pankajkoti** [comment]
hi @anai-s, the argument `virtualenv_dir ` in `ExecutionConfig` was added in PR #1079 . This PR was included in the latest release of `astronomer-cosmos==1.6.0`. So, we suggest you to please upgrade your Cosmos version to the latest released version 1.6.0 and try out. Looking forward to hear how it goes then.

**Comment by anai-s** [comment]
Hi @pankajkoti , thanks for the quick answer! It fixed my problem.
It would be cool if you specify the version compatibility as you did for the new feature [Source Nodes Rendering](https://astronomer.github.io/astronomer-cosmos/configuration/source-nodes-rendering.html) :)
